policy/modules/services/fail2ban.te

```diff
@@ -42,6 +42,8 @@ allow fail2ban_t self:fifo_file rw_fifo_file_perms;
 allow fail2ban_t self:unix_stream_socket { accept connectto listen };
 allow fail2ban_t self:tcp_socket { accept listen };
 
+corenet_tcp_bind_generic_node(fail2ban_t)
```

Comment:
Please move down with other corenet calls. Also, I don't see any corenet bind calls for the port(s), so there may still be missing permissions.

Reply:
Good point about possibly more missing permissions. I went looking through my audit logs to see what exactly triggered this, but couldn't find it. I think I'd prefer to drop this for now in that case.